src/validators/sub_folder.rs

```diff
@@ -0,0 +1,128 @@
+use crate::{Issue, IssueType, Severity};
+
+pub fn validate(raw_gtfs: &gtfs_structures::RawGtfs) -> Vec<Issue> {
+    let mut issues = vec![];
+    dbg!(&raw_gtfs.files);
+    if let Some(parent_folder) = raw_gtfs
+        .files
+        .iter()
+        .filter(|f| dbg!(f).ends_with("stops.txt"))
```

Comment:
Funny to test this only using a single file. I guess that's fine?!

Reply:
Nope, sounds good. We can adjust if it's a problem. Did you look how MobilityData checks this rule with their validator?

I'm wondering if some GTFS files (GTFS Flex) can have no `stops.txt` file.

Note: there is a `dbg!` statement here left.

Reply:
hum, they check all files https://github.com/MobilityData/gtfs-validator/blob/fdd6c9b03bc447664153fac344f63adc0a57a9fe/core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsInput.java#L114-L135

I think even GTFS-FLEX need a stops.txt, but I can check more files if you want (but I'll only be able to do it later this week). 

(oups :see_no_evil:  , sorry for the debug, I removed it)

Reply:
A single file should be good, targeting `stops.txt` seems the most obvious

Reply:
Sounds reasonable!